src/components/CEnqueue.vue

```diff
@@ -0,0 +1,182 @@
+              v-model="message.actorName"
+            >
+              <option
+                v-for="(actor, index) in declaredActors"
+                :key="index"
+                :value="actor"
+              >
+                {{ actor }}
+              </option>
+            </select>
+          </div>
+        </div>
+        <div class="w-full md:w-1/2 px-3">
+          <label
+            class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+            for="grid-last-name"
+          >
+            delay(min)
+          </label>
+          <input
+            class="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+            id="grid-last-name"
+            v-model="message.delay"
+            type="number"
+            min="0"
+          />
+          </div>
+        </div>
+        <div class="flex flex-wrap -mx-3 mb-2">
+          <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
+            <label
+              class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+              for="field_args"
+            >
+              Args
+            </label>
+            <textarea
+              class="appearance-none  h-40 block w-full bg-gray-200 focus:bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:border-gray-500"
+              id="field_args"
+              placeholder='[1, ["a", 7], "str"]'
+              v-model="message.args"
+            ></textarea>
+          </div>
+
+          <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
+            <label
+              class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+              for="field_kwargs"
+            >
+              Kwargs
+            </label>
+            <textarea
+              class="appearance-none  h-40 block w-full bg-gray-200 focus:bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:border-gray-500"
+              id="field_kwargs"
+              v-model="message.kwargs"
+              placeholder='{"nums":[1,2,5]}'
+            ></textarea>
+          </div>
+
+          <div class="w-full md:w-1/3 px-3 mb-6 md:mb-0">
+            <label
+              class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+              for="field_options"
+            >
+              Options
+            </label>
+            <textarea
+              class="appearance-none  h-40 block w-full bg-gray-200 focus:bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:border-gray-500"
+              id="field_options"
+              v-model="message.options"
+              placeholder='{"isOn":true}'
+            ></textarea>
+          </div>
+
+          <div class="flex">
+            <button
```

Comment:
add the button to the right, and add a button clear on it's left that remove data from all fields.